tests/unit/moduleapi/datatype2.tcl

```diff
@@ -79,6 +79,8 @@ start_server {tags {"modules"}} {
 
     test "datatype2: test aof rewrite" {
         r flushall
+        r config set appendonly yes
+        waitForBgrewriteaof r
```

Comment:
maybe in this test, instead of having AOF enabled on startup and then use BGREWRITEAOF, it is better to:
1. assert that `appendonly` is `no` at startup.
2. instead of calling `bgrewriteaof` and then `waitForBgrewriteaof`, we can just enable `appendonly` and then do `waitForBgrewriteaof` (i.e. all we want is a rewrite, so enabling AOF achieves that purpose too)